frontend/apps/web/app/(mgmt)/[account]/jobs/[id]/destinations/components/DestinationConnectionCard.tsx

```diff
@@ -159,10 +221,43 @@ export default function DestinationConnectionCard({
                     <FormDescription>
                       The location of the destination data set.
                     </FormDescription>
-                    <FormMessage />
+                    {/* <FormMessage /> */}
                   </FormItem>
                 )}
               />
+              {!isInitTableSchemaEnabled(
+                form.getValues('destinationOptions')
+              ) &&
+                tableErrors &&
+                tableErrors.length > 0 && (
```

Comment:
probably worth adding a loading state/skeleton here for larger databases where the validation may take a while.